--- a/Makefile	2019-07-16 08:30:17.953353053 -0400
+++ b/Makefile	2019-07-16 08:32:09.208540436 -0400
@@ -4,13 +4,13 @@
 build:
 	sed "s|MAKE_INSTALL_PREFIX = '.*'|MAKE_INSTALL_PREFIX = '$(PREFIX)'|g" blugon.py > blugon
 	gcc -std=c99 -O2 -I /usr/X11R6/include -o backends/scg/scg backends/scg/scg.c -L /usr/X11R6/lib -lm -lX11 -lXrandr
-	gzip --force --keep blugon.1
+	#gzip --force --keep blugon.1

 install:
 	install -d $(DESTDIR)$(PREFIX)/bin/
 	install -m755 blugon $(DESTDIR)$(PREFIX)/bin/blugon
 	install -d $(DESTDIR)$(PREFIX)/share/man/man1/
-	install -m644 blugon.1.gz $(DESTDIR)$(PREFIX)/share/man/man1/
+	install -m644 blugon.1 $(DESTDIR)$(PREFIX)/share/man/man1/
 	install -d $(DESTDIR)$(PREFIX)/share/bash-completion/completions/
 	install -m644 bash-completion/blugon $(DESTDIR)$(PREFIX)/share/bash-completion/completions/blugon
 	install -d $(DESTDIR)$(PREFIX)/lib/systemd/user/
@@ -34,4 +34,4 @@
 clean:
 	rm -f blugon
 	rm -f backends/scg/scg
-	rm -f blugon.1.gz
+	rm -f blugon.1
