--- a/Makefile	2019-05-22 21:13:35.318727618 -0400
+++ b/Makefile	2019-05-22 21:13:55.229727428 -0400
@@ -2,6 +2,8 @@
 
 VERCMD ?= git describe 2> /dev/null
 
+PREFIX = /usr
+
 NAME = ruler
 VERSION = $(shell $(VERCMD) || cat VERSION)
 YACC ?= yacc

--- a/man/Makefile	2019-05-22 16:00:58.050906502 -0400
+++ b/man/Makefile	2019-05-22 21:17:43.809725248 -0400
@@ -1,5 +1,7 @@
 include ../config.mk
 
+PREFIX = /usr
+
 MAN = ruler.1
 MANUAL = "Ruler Manual"
 ORG = "Ruler"
