diff --git a/meson.build b/meson.build
index f943211..9b718a4 100644
--- a/meson.build
+++ b/meson.build
@@ -198,9 +198,6 @@ custom_target(
   install: true,
   install_dir: join_paths(get_option('datadir'), 'terminfo'))
 
-install_data(
-  'LICENSE', 'README.md', 'CHANGELOG.md',
-  install_dir: join_paths(get_option('datadir'), 'doc', 'foot'))
 install_data(
   'foot.desktop', 'foot-server.desktop',
   install_dir: join_paths(get_option('datadir'), 'applications'))
