diff -u a/Makefile b/Makefile
--- a/Makefile	2019-10-05 06:59:34.000000000 -0400
+++ b/Makefile	2019-10-06 03:10:31.251128392 -0400
@@ -17,7 +17,7 @@

 RM       = rm -f

-prefix    ?= /usr/local
+prefix    ?= /usr
 bindir     = $(prefix)/bin
 libexecdir = $(prefix)/lib/jgmenu

@@ -27,7 +27,7 @@
 datarootdir= $(prefix)/share
 endif

-CFLAGS  += -g -Wall -Os -std=gnu99
+CFLAGS  += -g -Wall -std=gnu99
 CFLAGS  += -Wextra -Wdeclaration-after-statement -Wno-format-zero-length \
 	   -Wold-style-definition -Woverflow -Wpointer-arith \
 	   -Wstrict-prototypes -Wunused -Wvla -Wunused-result
