diff --git a/meson.build b/meson.build
index 200f2c7..63bf94f 100644
--- a/meson.build
+++ b/meson.build
@@ -14,8 +14,4 @@ if not meson.is_subproject()
                name : 'tllist',
                filebase : 'tllist',
                description : 'A C header file only implementation of a typed linked list')
-
-  install_data(
-    'LICENSE', 'README.md',
-    install_dir: join_paths(get_option('datadir'), 'doc', 'tllist'))
 endif
