--- a/src/Makefile
+++ b/src/Makefile
@@ -1,6 +1,6 @@
 debug ?= no
 static ?= no
-gzip_man ?= yes
+gzip_man ?= no
 # to get format compatible with GitHub archive use "gzip -S .gz" here
 compress_bin ?= bzip2

@@ -16,7 +16,6 @@
     suffix := .debug
 else
     ifeq ($(debug),no)
-        CXXFLAGS += -O3
         suffix := .opt
     else
         $(error debug should be either yes or no)
@@ -51,7 +50,7 @@
     LDFLAGS += -static -pthread
 endif

-PREFIX ?= /usr/local
+PREFIX ?= /usr
 DESTDIR ?= # root dir

 bindir := $(DESTDIR)$(PREFIX)/bin
