diff --git a/meson.build b/meson.build
index 499f1c2..1515a65 100644
--- a/meson.build
+++ b/meson.build
@@ -113,12 +113,6 @@ if not meson.is_subproject()
   pkg.generate(
     fcft_lib,
     description : 'Simple font loading and glyph rasterization library')
-
-  install_data(
-    'LICENSE', 'README.md', 'CHANGELOG.md',
-    install_dir: join_paths(get_option('datadir'), 'doc', 'fcft'))
-
-  subdir('doc')
 endif
 
 summary(
